src/controllers/statisticsController.js

```diff
@@ -0,0 +1,104 @@
+import models from '../db/models';
+
+const { Statistics } = models;
+
+/**
+ * @description class responsible for statistics of author's article readCount
+ *
+ * @class StatisticsController
+ */
+class StatisticsController {
+  /**
+   * @description - This method is responsible for counting updating article readCount
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} author - Username of the author of the article
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof StatisticsController
+   */
+  static async hasReadArticle(request, author) {
+    const articleSlug = request.params.slug;
+
+    const lastRead = await Statistics.findOne({
+      where: { articleSlug },
+      order: [['id', 'DESC']]
+    });
+
+    const readCount = 1;
+    if (!lastRead) {
+      await Statistics.create({
+        author,
+        articleSlug,
+        readCount
+      });
+    }
+    if (lastRead) {
```

Comment:
@marcdomain Nice work but wouldn't it be more logical to use an `else` statement here?

Reply:
Thanks for pointing this out, `else` would actually be good here. It will be adjusted.